app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts: list = command.split(maxsplit=2)
+
+    if len(parts) != 3 or parts[0] != "cp":
+        return
+
+    source_file: str = parts[1]
+    target_file: str = parts[2]
+
+    if source_file == target_file:
+        return
```

Comment:
Simplify and combine two conditions into one.
if source == target or command != "cp":
return